src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 16px;
```

Comment:
https://codeguide.academy/html-css.html#css-order rigth css order in all selectors